content/patterns/button/button-pattern.html

```diff
@@ -14,93 +14,94 @@
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          A <a class="role-reference" href="#button">button</a> is a widget that enables users to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.
-          A common convention for informing users that a button launches a dialog is to append &quot;&#8230;&quot; (ellipsis) to the button label, e.g., &quot;Save as&#8230;&quot;.
+          <a class="role-reference" href="#button">ボタン</a>は、フォームの送信、ダイアログの開閉、アクションのキャンセル、削除操作の実行など、利用者によるアクションやイベントをトリガするウィジェットです。
+          ボタンがダイアログを起動することを利用者に知らせる一般的な慣例は、ボタンのラベルに「…」（省略記号）を追加することです。例：「名前を付けて保存…」。
         </p>
-        <p>In addition to the ordinary button widget, WAI-ARIA supports 2 other types of buttons:</p>
+        <p>通常のボタンウィジェットに加えて、WAI-ARIA は他の 2 種類のボタンをサポートしています。</p>
         <ul>
           <li>
-            Toggle button: A two-state button that can be either off (not pressed) or on (pressed).
-            To tell assistive technologies that a button is a toggle button, specify a value for the attribute <a href="#aria-pressed" class="state-reference">aria-pressed</a>.
-            For example, a button labelled mute in an audio player could indicate that sound is muted by setting the pressed state true.
-            <strong>Important:</strong> it is critical the label on a toggle does not change when its state changes.
-            In this example, when the pressed state is true, the label remains &quot;Mute&quot; so a screen reader would say something like &quot;Mute toggle button pressed&quot;.
-            Alternatively, if the design were to call for the button label to change from &quot;Mute&quot; to &quot;Unmute,&quot; the aria-pressed attribute would not be needed.
+            トグルボタン: オフ（押されていない）又はオン（押されている）のいずれかの 2 状態ボタンです。
+            ボタンがトグルボタンであることを支援技術に伝えるには、<a href="#aria-pressed" class="state-reference">aria-pressed</a> 属性の値を指定します。
+            たとえば、オーディオプレーヤーのミュートとラベル付けされたボタンは、押された状態を true に設定することで、サウンドがミュートされていることを示すことができます。
+            <strong>重要:</strong> トグルの状態が変化しても、ラベルが変化しないことが重要です。
+            この例では、押された状態が true の場合でもラベルは「ミュート」のままであるため、スクリーンリーダーは「ミュートトグルボタンが押されました」のように読み上げます。
+            あるいは、デザインでボタンのラベルを「ミュート」から「ミュート解除」に変更する必要がある場合、aria-pressed 属性は必要ありません。
           </li>
-          <li>Menu button: as described in the <a href="../menu-button/menu-button-pattern.html">menu button pattern</a>, a button is revealed to assistive technologies as a menu button if it has the property <a href="#aria-haspopup" class="property-reference">aria-haspopup</a> set to either <code>menu</code> or <code>true</code>.</li>
+          <li>メニューボタン: <a href="../menu-button/menu-button-pattern.html">メニューボタンパターン</a>で説明されているように、<a href="#aria-haspopup" class="property-reference">aria-haspopup</a> プロパティが <code>menu</code> 又は <code>true</code> に設定されている場合、ボタンは支援技術にメニューボタンとして表示されます。</li>
```

Comment:
> ボタンは支援技術にメニューボタンとして表示されます。

「表示」は「ユーザーから視覚的に見えるかたちになる」をイメージしてしまいそう。実際は支援技術は視覚的な手段だけでないのはもちろん、「is revealed」されたものがユーザーに提示されるとも限らない。

「表示されます」は「提示されます」のほうがいいかも。あくまで支援技術に対してどういうかたちで情報が伝わるかというだけなので。

```suggestion
          <li>メニューボタン: <a href="../menu-button/menu-button-pattern.html">メニューボタンパターン</a>で説明されているように、<a href="#aria-haspopup" class="property-reference">aria-haspopup</a> プロパティが <code>menu</code> 又は <code>true</code> に設定されている場合、ボタンは支援技術にメニューボタンとして提示されます。</li>
```